Client-Server/Client/SeTCPClient.m

```diff
@@ -0,0 +1,117 @@
+#import "SeTCPClient.h"
+#import <Foundation/Foundation.h>
+#import <CoreFoundation/CoreFoundation.h>
+
+#include <arpa/inet.h>
+
+
+@interface SeTCPClient()
+
+- (void)sendData:(NSTimer *)timer;
+
+@property (nonatomic) CFSocketRef socket;
+@property (nonatomic) CFDataRef addrRef;
+@property (nonatomic) NSString *msg;
+@property (nonatomic) NSString *addr;
+@property (nonatomic) NSUInteger port;
+
+@end
+
+@implementation SeTCPClient
+
+static const double msgTimeInterval = 1.0f;
+
+- (id)init {
+    return [self initWithMessage:@"lol" toAddress:@"127.0.0.1" andPort:6666];
+}
+
+- (id)initWithMessage:(NSString *)message toAddress:(NSString *)addr andPort:(NSUInteger)port {
+    if ((self = [super init])) {
+        _msg = message;
+        _addr = addr;
+        _port = port;
+    }
+    
+    return self;
+}
+
+static void onConnect(CFSocketRef s, CFSocketCallBackType type, CFDataRef address, const void *data, void *info) {
```

Comment:
Define this outside the class, it's a C function and doesn't belong to the class